InteractionBase/src/handlers/GenericHandler.cpp

```diff
@@ -109,6 +109,11 @@ ActionPrompt* GenericHandler::actionPrompt_{};
 QPoint GenericHandler::cursorOriginMidPoint_;
 GenericHandler::CursorMoveOrientation GenericHandler::cursorMoveOrientation_ = NoOrientation;
 
+QHash<int, QPoint> GenericHandler::keyToIndexMap_
```

Comment:
Now that this is in GenericHandler, we should give it a more specific name. E.g. `keyToViewItemIndexMap_`